app/migrations/seed_dev.py

```diff
@@ -0,0 +1,162 @@
+    if instance:
+        return instance
+    else:
+        instance = Organisation(name=name, token=token)
+
+        db.session.add(instance)
+        return instance
+
+
+def get_or_create_admin_user(email, new_organisation):
+    instance = User.query.execution_options(show_all=True).filter_by(
+        email=str(email).lower()).first()
+    if instance:
+        return instance
+    else:
+        user = User()
+        user.create_admin_auth(email=email,
+                               password='TestPassword', tier='sempoadmin')
+        user.organisations.append(new_organisation)
+        user.is_activated = True
+        return user
+   
+
+def get_or_create_user(email, business_usage_id):
+    instance = User.query.execution_options(show_all=True).filter_by(
+        email=str(email).lower()).first()
+    if instance:
+        return instance
+    else:
+        instance = User(email=email, business_usage_id=business_usage_id)
+
+        db.session.add(instance)
+        return instance
+
+
+def get_or_create_transer_account(name, blockchain_address, organisation):
+    instance = TransferAccount.query.filter_by(name=name).first()
+    if instance:
+        return instance
+    else:
+        instance = TransferAccount(
+            blockchain_address=blockchain_address, organisation=organisation)
+
+        instance.name = name
+        db.session.add(instance)
+        return instance
+
+
+def randomString(stringLength=10):
+    """Generate a random string of fixed length """
+    letters = string.ascii_lowercase
+    return ''.join(random.choice(letters) for i in range(stringLength))
+
+
+def create_users_different_transer_usage(wanted_nr_users):
+    i = 1
+    user_list = []
+    transer_usages_ids = TransferUsage.query.with_entities(
+        TransferUsage.id).all()
+    while i < wanted_nr_users:
+        random_usage_id = random.choice(transer_usages_ids)[0]
+        new_user = get_or_create_user(
+            'user-nr-' + str(i) + '@test.com', random_usage_id)
+        new_transer_account = get_or_create_transer_account(
+            'transfer-account-nr-' + str(i), blockchain_address, new_organisation)
+
+        if len(new_user.transfer_accounts) < 1:
+            new_user.transfer_accounts.append(new_transer_account)
+        user_list.append(new_user)
+        i += 1
+    return user_list
+
+
+def create_transfers(sender, user_list, wanted_nr_transfers):
+    transfer_list = []
+    i = 0
+    while i < wanted_nr_transfers:
+        transfer = CreditTransfer(
+                                    i,
+                                    sender_user=user1,
+                                    recipient_user=random.choice(user_list),
+                                    token=token,
+                                    uuid=randomString())
+        db.session.add(transfer)
+        transfer_list.append(transfer)
+        i += 1
+    return transfer_list
+
+
+if __name__ == '__main__':
```

Comment:
So I just added seed for all the stuff that was needed to test the get_most_relevant_transfer_usage function and an admin user. If anyone has other suggestion I can add it to this pr or create a new pr for that later